### PR TITLE
Feature/date override

### PR DESCRIPTION
**Pull request by david-potgieter**
I needed an option where I could specify the published date for older posts. This was added to the API: [comment](https://github.com/Medium/medium-api-docs/issues/6#issuecomment-180459596)

Unfortunately there are some limits: you can only override the date on stories that are directly set to PUBLIC so added additional params for this too.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const fs = require('fs');
 const path = require('path');
 const lodash = require('lodash');
 const mediumImport = require('./lib/mediumImport');
+const moment = require('moment');
```
Comment (ryyppy) [inline index.js:9]: hmmm, do we really need to pull in moment for that?
I would rather have something more lightweight, like `date-fns` or something?

**Comment by ryyppy** [comment]
@david-potgieter Really cool! I didn't expect too many people to use this tool... so I trust you that this feature will work as expected :-)

I am just not a very huge fan of moment, since we don't want to pull in the whole universe just for some basic function... maybe we can either use date-fns or even use some native ISO interface from node?

Would be happy to merge and republish when we figured that out!

**Comment by david-potgieter** [comment]
Agreed! Wanted to extend the CLI to include date formats - hence the heavy moment include. But will play around a little over the weekend.

**Comment by ryyppy** [comment]
Cool, just let me know if there is any update / problem 👍

**Pull request closed without merging**